Makefile

```diff
@@ -0,0 +1,68 @@
+.PHONY: build-linux64-static build-linux64-static-tar build-docker-image \
+	build-docker-image-tar build-local build-local-tar build-all \
+	release-candidate print-vars
+
+NAME := $(shell cat package.yaml | grep "name:" | sed 's/name: *//g')
+VERSION := $(shell cat package.yaml | grep "version:" | sed 's/version: *//g')
+DIST := $(PWD)/dist
+OS_NAME := $(shell uname -s | tr '[:upper:]' '[:lower:]')
+OS_ARCH := $(shell uname -m)
+
+BUILD_NUMBER := $(shell scripts/nextbuildnumber.sh ${VERSION})
+FULL_VERSION := ${VERSION}-${BUILD_NUMBER}
+DOCKER_TAG := ${NAME}:${FULL_VERSION}
+
+OUTPUT_DIR := ${NAME}-${FULL_VERSION}-${OS_NAME}-${OS_ARCH}
+FULL_OUTPUT_DIR := ${DIST}/${OUTPUT_DIR}
+
+LINUX64_OUTPUT_DIR := ${NAME}-${FULL_VERSION}-linux-x86_64-static
+LINUX64_FULL_OUTPUT_DIR := ${DIST}/${LINUX64_OUTPUT_DIR}
+
+LINUX64_STATIC_TAR := ${DIST}/${LINUX64_OUTPUT_DIR}.tar.gz
+LOCAL_TAR  := ${DIST}/${OUTPUT_DIR}.tar.gz
+DOCKER_IMAGE_TAR := ${DIST}/${NAME}-${FULL_VERSION}-docker.tar.gz
+
+build-linux64-static:
+	mkdir -p ${LINUX64_FULL_OUTPUT_DIR}
+	docker run --rm \
+		-v $(PWD):/usr/src/build \
+		-v $(HOME)/.stack:/root/.stack \
+		-v ${LINUX64_FULL_OUTPUT_DIR}:/root/.local/bin \
+		-w /usr/src/build -it itkach/alpine-haskell-stack19 \
+		bash -c "stack config set system-ghc --global true && \
+			 stack install --ghc-options='-optl-static -fPIC -optc-Os'"
+
+build-linux64-static-tar: build-linux64-static
+	tar -C ${DIST} -czvf ${LINUX64_STATIC_TAR} ${LINUX64_OUTPUT_DIR}
+
+build-docker-image: build-linux64-static
+	docker build . --tag ${NAME} --tag ${DOCKER_TAG}
+
+build-docker-image-tar: build-docker-image
+	docker save ${DOCKER_TAG} | gzip --best -c > ${DOCKER_IMAGE_TAR}
+
+build-local:
```

Comment:
How about a `build-osx` that checks that you're on OSX before running `build-local`?

Reply:
I had the same thought, but then it wasn't immediately clear to me how to implement in the Makefile so I thought about it some more and decided it doesn't seem to be solving any actual problem :) I'd rather skip it for now